pkg/asset/machines/master.go

```diff
@@ -144,7 +145,20 @@ func masterPool(pools []types.MachinePool) types.MachinePool {
 	return types.MachinePool{}
 }
 
-func listFromMachines(objs []clusterapi.Machine) *metav1.List {
+func listFromMachines(objs []machineapi.Machine) *metav1.List {
```

Comment:
why this change.. ?

Reply:
becomes machines now belong to machine.openshift.io API group, hence the `machineapi.Machine` type

Reply:
Sorry wrong line, more as to why keep the deprecated one too? It that bring used?

Reply:
@abhinavdahiya we are providing a backward compatible pivoting path for any actuator [openshift/machine-api-operator#192](https://github.com/openshift/machine-api-operator/pull/192) This PR finalise the pivot for aws. We'll put a follow up for libvirt right away and once the openstack actuator is updated to rely on machine.openshift.io and they put a PR analogous to this one for Opensatck we will fully drop support for `cluster.k8s.io` either here and on the machine api operator [openshift/machine-api-operator#195](https://github.com/openshift/machine-api-operator/pull/195)